package.json

```diff
@@ -8,9 +8,12 @@
     "compile": "npm run compile:css && npm run compile:js",
     "dev:css": "sass --load-path=node_modules --watch sass/etna.scss:templates/static/css/dist/etna.css",
     "dev:js": "webpack --config webpack.config.js --mode=development --watch",
+    "dev": "npm run dev:css & npm run dev:js",
```

Comment:
For convenience: allow watching of the js and css at the same time

Reply:
See here: https://github.com/nationalarchives/ds-wagtail/pull/1340/files#r1347244698

Reply:
We don't even use the `dev` command - I think it needs to go. We run this script from inside the dev container: https://github.com/nationalarchives/ds-wagtail/blob/develop/dev/bin/dev

Reply:
I added it as I couldn't see a way to compile the css and js at the same time, which seemed odd - I'm used to running these commands on my local machine - is there a way to run them both together in the dev container then?

Reply:
To compile and watch both you can just use two terminals or run one in the background like you have here with the `&` but I don't think there was a huge need to jump back and forth between developing the CSS and JavaScript without utilising what the `dev` container has to offer.